elm-book/elm.json

```diff
@@ -37,7 +37,8 @@
             "pablohirafuji/elm-syntax-highlight": "3.4.1",
```

Comment:
bro out of curiosity we still need this package? totally unrelated with the PR but i'm just curious

Reply:
I was really confused on why we had that, until I realized it's in elm-book 🤣

We only need it as an indirect dependency for elm-book to work. I thought we were already running elm-review on elm-book, but seems like that's not the case.

I could fix it on this PR, but I ran elm-review on elm-book and uh...
![image](https://user-images.githubusercontent.com/39280468/184404737-dc118bc5-5f49-47a6-bef2-0b2312ed16df.png)

I think it's better to have another PR to deal with it 😅 I created #809 for that